### ISSUE TITLE
Create getting started tutorial

### ISSUE THREAD

**Issue by aulia-adil** [SasanLabs/owasp-zap-jwt-addon#41]
I'm new to ZAP and I'm having a problem understanding this plug-in. 

Therefore, it would be nice if I knew the getting started tutorial instead of trying to understand the configuration.

I would love to create the tutorial if it could be included here.

**Comment by preetkaran20** [comment]
Yes, perfect, please let me know if you need any help on understanding this plugin

**Comment by marvi3** [comment]
I would also really like to use this plugin but I am having problems understanding it.
The provided tutorial is not helping me much. As I understood, this plugin is attacking the JWT authentication mechanism.
However I do not understand how to set up the configuration, what to fill in.
